### ISSUE TITLE
archive.monarchinitiative.org does not exist

### ISSUE THREAD

**Issue by TomConlin** [monarch-initiative/monarch-devops#47]
We annotate our datasets with their "forever home"  in. 
https://archive.monarchinitiative.org/ttl/
I have not seen this resource and assume it was lost in the move up from California.

I do not know how It was organized originally.  My preference now would be: 

https://archive.monarchinitiative.org/YYYYMMDD/everything
[dev/ dot/ golr/ neo4j/ owl/ pub/ service-scripts/ ttl/]

**Comment by jmcmurry** [comment]
Yessssssplease. :)

**Comment by vanmatre** [comment]
Which server should archive point to (as a cname)?

**Comment by TomConlin** [comment]
Looks like this issue has previous tickets (I should have checked) 
https://github.com/monarch-initiative/monarch-devops/issues/3
https://github.com/monarch-initiative/monarch-devops/issues/27

**Comment by jnguyenx** [comment]
@TomConlin I have never seen it working. Each time Nicole tried to show me that, it was unavailable. Do you have any idea how it was implemented? Was it just a cron job? If yes what was the frequency?

**Comment by TomConlin** [comment]
There are triples created in the ingest process which reference http://archive.monarchinitiative.org/ttl,  I have not found more.
As I see it, our  data/ttl  etc. directories should just be a symlinks to the latest archive.

**Comment by kshefchek** [comment]
@vanmatre we would want archive.monarchinitiative.org and data.monarchinitiative.org on separate physical machines.  Other than that the vm would be similar to monarch-ttl-prod.

@TomConlin also suggested we consider a second archive at one of our partner sites but we can break that out to a separate ticket once the OHSU archive is set up.

**Comment by mellybelly** [comment]
pinging @cmungall so he knows this is here.
Especially with new R24 and Translator, we need to get right with our archiving.

**Comment by jmcmurry** [comment]
@vanmatre Any update on this?

**Comment by vanmatre** [comment]
To verify:

 This is a request to create a new VM named archive.monarchinitiative.org sized like monarch-ttl-prod?

**Comment by TomConlin** [comment]
This is a request for a VM named archive.monarchinitiative.org not sharing hardware with monarch-ttl-prod.

monarch-ttl-prod now has:

    tomc@monarch-ttl-prod:~$ df -lh
    Filesystem      Size  Used Avail Use% Mounted on
    udev            2.0G  4.0K  2.0G   1% /dev
    tmpfs           396M  444K  395M   1% /run
    /dev/vda3       122G  102G   14G  88% /
    none            4.0K     0  4.0K   0% /sys/fs/cgroup
    none            5.0M     0  5.0M   0% /run/lock
    none            2.0G     0  2.0G   0% /run/shm
    none            100M     0  100M   0% /run/user
    /dev/vda1       232M  100M  116M  47% /boot

since this is working now, it must be sufficient. 
However because of the nature of archiving I expect  archive.monarchinitiative.org's storage needs to grow more quickly than  monarch-ttl-prod's.  
So whatever makes the most sense with that factored in is what we want.

**Comment by vanmatre** [comment]
archive is active (address 10.95.40.128) and is in the Monarch internal DNS. 

Let me know if you need external (to the monarch servers) access to the machine and I will get the process going to assign an external address. Also if so I will need which ports to have opened..

**Comment by TomConlin** [comment]
The outside world does need http/https access to archive.monarchinitiative.org 
inside world should be able to ssh/scp

**Comment by vanmatre** [comment]
OK, I'll get the request in to our netops folks. Takes a few days.

**Comment by TomConlin** [comment]
That is great Ethan, thanks.

**Comment by vanmatre** [comment]
You are welcome.

**Comment by vanmatre** [comment]
Archive now has an external address and has been added to the external DNS namespace.

It appears that http and https are open but archive is refusing connections.

**Comment by TomConlin** [comment]
I am not sure archive has a http server installed yet. (do not think it does)

I was guessing we would go with nginx and data would live in   

    /var/www/data/<release_datestamp>/
	analysis  
	dev  
	dipper  
	dot  
	exomiser  
	golr  
	owl  
	previous        <**** will be archives initial contents  
	pub  
	service-scripts  
	ttl

**Comment by jnguyenx** [comment]
You have to install nginx yourself, you can refer to the ansible script from monarch-ttl-prod.

**Comment by TomConlin** [comment]
archive.monarchinitiative.org  is live
http redirects to https (letsencrypt)
I did not make a https cert for www.archive. because it never existed
404's redirect to the root because we have produced a number of years worth of IRI 
to files with embedded arbitrary dates (to the minute) which I do not have. 

future IRI should look like  https://archive.monarchinitiative.org/201702/ttl/
which should work till we jump to weekly releases.
